auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -188,22 +215,7 @@ public static AuthUI getInstance(@NonNull FirebaseApp app) {
         synchronized (INSTANCES) {
             authUi = INSTANCES.get(app);
             if (authUi == null) {
-                authUi = new AuthUI(app);
-                authUi.mAuth.setFirebaseUIVersion(BuildConfig.VERSION_NAME);
-                INSTANCES.put(app, authUi);
-            }
-        }
-        return authUi;
-    }
-
-    @RestrictTo(RestrictTo.Scope.TESTS)
-    static AuthUI getTestInstance(FirebaseApp app) {
-        AuthUI authUi;
-        synchronized (INSTANCES) {
-            authUi = INSTANCES.get(app);
-            if (authUi == null) {
-                authUi = new AuthUI(app);
-                INSTANCES.put(app, authUi);
+                INSTANCES.put(app, authUi = new AuthUI(app));
```

Comment:
nit: I find the old multi-line version much easier to understand.  I don't often see Java devs use the value of an assignment expression.

Reply:
SGTM